admin/articles.rb

```diff
@@ -404,7 +404,7 @@
         )
       end
     end
-    render json: Oj.dump(articles: articles)
+    render json: Oj.dump("articles" => articles)
```

Comment:
`Oj.dump({articles: articles}, mode: :compat)` would be the same fix but would be 1.4x slower?

Even ist is slower, I would prefer `Oj.dump({articles: articles}, mode: :compat)` over `Oj.dump("articles" => articles)`